quinn-proto/src/endpoint.rs

```diff
@@ -1270,7 +1268,7 @@ impl Incoming {
 impl fmt::Debug for Incoming {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         f.debug_struct("Incoming")
-            .field("addresses", &self.addresses)
+            .field("network_path", &self.network_path)
```

Comment:
nit: we should probably start migrating to derive_more now. but feel fee to not do that either.